clients/pkg/promtail/client/client_test.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/grafana/loki/clients/pkg/promtail/api"
 	"github.com/grafana/loki/clients/pkg/promtail/utils"
+
```

Comment:
Change caused by golangci-lint settings (see `.golangci.yml`), which does not count `github.com/grafana/loki/clients/*` as "local" packages:

```
linters-settings:
  goimports:
    local-prefixes: github.com/grafana/loki/pkg,github.com/grafana/loki/tools
```